tracks/ruby/exercises/high-scores/mentoring.md

```diff
@@ -27,44 +25,38 @@ class HighScores
     scores.sort.reverse.take(3) 
   end
 
-  # composing a string either with string interpolation or with an if.. else.. statement
   def report
     difference = "#{personal_best - latest} short of" unless personal_best == latest
 
-    "Your latest score was #{latest}. That's #{difference} your personal best!".squeeze
+    "Your latest score was #{latest}. That's #{difference} your personal best!".squeeze(' ')
   end
 end
 
 ```
-
-### Reasonable approaches
-
-- See the one above
-- Defining `@scores`, `@personal_best` and `personal_top` in the constructor.  
+### Reasonable variants
+ - Using a getter method `scores`, not `attr_reader`: That's okay at this point in the track, no need to suggest `attr_reader`.
+ - Variants in composing the string in `report`, as long as it's not a complicated if/else statement (see Talking Points).
 
 ### General notes
 - The Instructions point to a beginner friendly explanation of instantiating a class: 
 http://ruby-for-beginners.rubymonstas.org/writing_classes/initializers.html
 and to the rubydocs Array. 
-- Most submissions use an getter method `scores`, not `attr_reader`. 
-That's okay at this point in the track, no need to suggest `attr_reader`.
 - Beginners create the attribute reader, but may call the instance variable instead of the getter. 
 This link gives a nice introduction of getters and setters, and the naming convention in Ruby:
-https://launchschool.com/books/oo_ruby/read/classes_and_objects_part1#accessormethods.
+[launchschool: accessor_methods](https://launchschool.com/books/oo_ruby/read/classes_and_objects_part1#accessormethods).
 It's recommended to link to this or your favorite explanation, instead of typing the solution for them.
 - `sort.reverse` is not very performant, but at this point in the track it's more important that students explore the 
-standard lib (and work with methods chaining) than that they know how to optimize.
+standard libraries (and work with method chaining) than that they know how to optimize.
```

Comment:
_core library_ rather than the standard library.  The standard library is probably not where we want them to go.

Reply:
@kotp See this comment here in case you missed it: https://github.com/exercism/website-copy/pull/633#discussion_r240696982

I think "general" is a better word when not refering to "core" or "standard" specifically.

Reply:
I agree, it removes that stroop effect completely.  Still better to get the changes in, I think, than be stopped by that.  Not a strong objection, I just see it causing (me) cognitive issues. :stuck_out_tongue_winking_eye: